src/main/java/com/it4045/pokemonviewer/enterprise/PokemonViewerController.java

```diff
@@ -47,4 +50,14 @@ public ResponseEntity getPokemon() {
             return new ResponseEntity(HttpStatus.INTERNAL_SERVER_ERROR);
         }
     }
+
+    @DeleteMapping("/pokemons")
+    public ResponseEntity deletePokemon(@PathVariable("id") int id)    {
+        try {
+            pokemonService.delete(id);
+            return new ResponseEntity(HttpStatus.OK);
+        }   catch (Exception e) {
+            return new ResponseEntity(HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
```

Comment:
Added @DeleteMapping to PokemonViewerController.java. @DeleteMapping annotation is for mapping HTTP Delete requests into particular handler method. I have also used try catch method in this. This will help the user to delete any pokemon which they does not want using id.